website/docs/cloud-docs/policy-enforcement/manage-policy-sets.mdx

```diff
@@ -12,7 +12,7 @@ Policies are rules that Terraform Cloud enforces on Terraform runs. You can defi
 @include 'tfc-package-callouts/policies.mdx'
 <!-- END: TFC:only name:pnp-callout -->
 
-Policy sets are collections of policies you can apply globally or to specific [projects](/terraform/cloud-docs/workspaces/organize-workspaces-with-projects) and workspaces in your organization. For each run in the selected workspaces, Terraform Cloud checks the Terraform plan against the policy set. Depending on the [enforcement level](#policy-enforcement-levels), failed policies can stop a run in a workspace.
+Policy sets are collections of policies you can apply globally or to specific [projects](/terraform/cloud-docs/workspaces/organize-workspaces-with-projects) and workspaces in your organization. For each run in the associated workspaces, Terraform Cloud checks the Terraform plan against the policy set. Depending on the [enforcement level](#policy-enforcement-levels), failed policies can stop a run in a workspace. You can exclude selected workspaces from a policy set if you don't want those workspaces' runs to be checked against the policy set.
```

Comment:
"associated" didn't seem like quite the right word here, but I wanted something that meant essentially "For each run in the workspaces that the policy set applied to" -- e.g.:

global scoped -> all workspaces in the org
project scoped -> all workspaces in the project
workspace scoped -> all workspaces scoped directly

Reply:
I think we've been using "attached" as the verb for "attaching a policy set to a project". also a small suggestion for active voice:

```suggestion
Policy sets are collections of policies you can apply globally or to specific [projects](/terraform/cloud-docs/workspaces/organize-workspaces-with-projects) and workspaces in your organization. For each run in the attached workspaces, Terraform Cloud checks the Terraform plan against the policy set. Depending on the [enforcement level](#policy-enforcement-levels), failed policies can stop a run in a workspace. You can exclude selected workspaces from a policy set if you don't want Terraform Cloud to check those workspaces' runs against the policy set.
```

Reply:
Thanks! I was trying to avoid "attached" because the sentence refers not just to runs on workspaces attached to the policy sets, but also runs in workspaces connected indirectly to a policy set (i.e. if the policy set is attached to the workspace's project or the policy set is global). Definitely a nit pick, so I'm happy to go back to "attached".

Reply:
Updated in bbdf764!